src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/CommunityOffenderManagerRepository.kt

```diff
@@ -7,5 +7,6 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.CommunityOff
 @Repository
 interface CommunityOffenderManagerRepository : JpaRepository<CommunityOffenderManager, Long> {
   fun findByStaffIdentifier(staffIdentifier: Long): CommunityOffenderManager?
+  fun findByStaffIdentifierOrUsername(staffIdentifier: Long, username: String): List<CommunityOffenderManager>?
```

Comment:
think this will always be a non-null list - just empty if no matches

Reply:
Thanks for the info